panther_lights/launch/lights.launch.py

```diff
@@ -58,26 +59,31 @@ def generate_launch_description():
         description="Path to a YAML file with a description of the user defined animations.",
     )
 
-    lights_driver_node = Node(
-        package="panther_lights",
-        executable="driver_node",
-        name="lights_driver",
+    lights_container = ComposableNodeContainer(
+        package="rclcpp_components",
+        name="lights_container",
         namespace=namespace,
-        remappings=[("/diagnostics", "diagnostics")],
-        emulate_tty=True,
-        on_exit=Shutdown(),
-        condition=UnlessCondition(use_sim),
-    )
-
-    lights_controller_node = Node(
-        package="panther_lights",
-        executable="controller_node",
-        name="lights_controller",
-        parameters=[
-            {"led_config_file": led_config_file},
-            {"user_led_animations_file": user_led_animations_file},
+        executable="component_container",
+        composable_node_descriptions=[
+            ComposableNode(
+                package="panther_lights",
+                plugin="panther_lights::DriverNode",
+                name="lights_driver",
+                namespace=namespace,
+                remappings=[("/diagnostics", "diagnostics")],
+                condition=UnlessCondition(use_sim),
+            ),
+            ComposableNode(
+                package="panther_lights",
+                plugin="panther_lights::ControllerNode",
+                name="lights_controller",
```

Comment:
I know this is not an objective of this task but I've spotted the debris after node renaming in file `led_animations_queue.cpp` line 121. Could you change it?